examples/DrawingInTheAir/DrawingInTheAir_MotionActivated

```diff
@@ -0,0 +1,279 @@
+/*
+ * This example demonstrates using the pattern matching engine (CuriePME)
+ * to classify streams of accelerometer data from CurieIMU.
+ *** ENHANCEMENT: Automatic data capture with build in MOTION DETECTION functionality, no need of button or any extra input hardware.
```

Comment:
This line has a lot of unnecessary words and capitalization. Please replace it with something *simple and clear*

"Motion is automatically detected using the CurieIMU motion detection interrupt"